source/includes/steps-install-shell-linux-rpm.yaml

```diff
@@ -50,7 +50,7 @@ content: |
 
                [mongodb-org-{+mdb-version+}]
                name=MongoDB Repository
-               baseurl=https://repo.mongodb.org/yum/amazon/$releasever/mongodb-org/{+mdb-version+}/$basearch/
+               baseurl=https://repo.mongodb.org/yum/amazon/2023/mongodb-org/{+mdb-version+}/$basearch/
```

Comment:
It seems like right now this is a template for all AL versions, we should make clear that the user needs to adjust this value depending on the actual version on their system

Reply:
My thought is that the manual instructions underneath (starting with "You can also download...") indicate that the users can switch depending on their version. Do you think it should be emphasized more?

Reply:
Yeah, I think so, the manual downloads don't really cover the same use case, and we already know that users don't have a particularly easy time figuring out how to pick the "right" way to install the MongoDB server through apt/yum.

Reply:
For what it's worth, you don't need to consider this a blocking issue, I'm just worried that this will lead to users accidentally using the 2023 repo when they didn't intend to do that, maybe that's just fine and maybe AL2023 is already widely adopted enough that it doesn't really matter

Reply:
I just added a note under the .repo file about Amazon Linux 2 - what do you think?